pkg/controllers/interruption/metrics.go

```diff
@@ -34,9 +35,9 @@ var (
 			Namespace: metrics.Namespace,
 			Subsystem: interruptionSubsystem,
 			Name:      "received_messages",
-			Help:      "Count of messages received from the SQS queue. Broken down by message type and whether the message was actionable.",
```

Comment:
I'm thinking the only metric here that _really_ makes sense to add the provisioner label is the `actionsPerformed` metric. Every other metric has to deal with SQS messages generally so we don't really have context-specific information about the provisioner in there.

Reply:
That makes sense, I agree - I'll remove